packages/services/f-braze-adapter/src/services/contentCard.service.js

```diff
@@ -127,15 +127,22 @@ class ContentCards {
      * There's a bug in Braze where duplicate content cards end up being displayed twice
      * This is being fix so in the mean time this function removes cards that have the same title and
      * card type it also orders the cards by the last updated date.
+     *
+     * The ternary ensures the right predicate for findIndex is used, to ensure that cards with deduplicationKeys are
+     * not deduped by cards without deduplication keys, or vice-versa
+     *
      * @property {Object[]} this.cards
      * @returns {ContentCards}
      */
     removeDuplicateContentCards () {
-        this.cards = orderBy(this.cards, 'updated').filter((contentCard, index, item) => index ===
-            findIndex(
-                item,
-                card => card.title === contentCard.title &&
-                    card.type === contentCard.type
+        this.cards = orderBy(this.cards, ['updated'], ['desc'])
```

Comment:
This file was only left in for reference (pretty sure)

Reply:
@matthewhardern - yes, I think it was applied automatically by the rebase. Is it worth removing the file entirely?

Reply:
I can remove it at a later date happy for this PR to be as is.